### ISSUE TITLE
Accessibility: don't trap the tab key in the task editor's contents textfield

### ISSUE THREAD

**Issue by nekohayo** [getting-things-gnome/gtg#1364]
There are many accessibility problems in GTG, but here is an easy one to fix: the task editor's main contents textview widget has tabs insertion enabled, so it effectively traps you into that widget; you cannot navigate away from it using the keyboard, so for blind users it is pretty much impossible to access the toolbar above it with `Shift+Tab` (or `Ctrl+Shift+Tab`) for example.
